apps/slackbot/bolt_app.py

```diff
@@ -16,6 +16,8 @@
 from tools import get_tools
 from vectorstores import ConversationStore, LocalChromaStore
 
+VERBOSE_DEFAULT = True  # set the verbose default behaviour
```

Comment:
Make this an environment variable in `.env-sample` and `config.py`.

Reply:
I also agree. But this would require the main devs to change the deployed .env. Also, this variable would be hidden inside .env, which is different for every local environment. I'll leave it as is, and @saminegash or @20001LastOrder can make this change they want.